docs/source/DebuggingWithGDB.rst

```diff
@@ -185,3 +181,38 @@ breakpoints, executing commands and querying for output.
 
 When you check the output (``--show-job-log``) you can see that despite
 declaring the variable as 0, ff is injected and printed instead.
+
+
+rr wrapper
+----------
+
+It uses the ``--wrapper`` avocado functionality to run matching binaries
+inside the specified script. There is an example `rr` script in
+``/usr/share/avocado/wrappers/rr.sh`` which stores the information in
+test's `output_dir/rr` as `$bin_name.$serial_id`.
+
+Let's have a look at an example with malfunctioning qemu. Run the
+malfunctioning qemu inside avocado with wrapper::
+
+    $ avocado run "/usr/local/bin/qemu-system-aarch64 -machine virt" --wrapper /usr/share/examples/wrappers/rr.sh:*qemu*
```

Comment:
@ldoktor @adereis the amount of comments suggest that we don't know what we don't know, that is, it's just too early to try to formalize what it means to support `rr` in avocado.  Let's wait for the dust to settle and properly understand the use possible use cases and then come up with examples that users may really relate to.

Lukas: I don't see a problem in mentioning in the release notes that we've added an example wrapper for `rr`.  Wrappers are already documented, so the "right" users will have no trouble using it. 

I'll commit the bug fix only for now.